Macros/ACS_summary_geo_source_tr_vars.sas

```diff
@@ -5363,6 +5366,293 @@
+		
+
+%if &_last_year. >= 2023  %then %do;
+
+				mortcstbrden_&_years.= sum(b25140be3, b25140ie3, b25140he3, b25140ce3, b25140fe3, b25140ge3, b25140de3, b25140ee3);
+				mortsvrecstbrden_&_years.= sum(b25140be4, b25140ie4, b25140he4, b25140ce4, b25140fe4, b25140ge4, b25140de4, b25140ee4);
+				mortcstbrdencalc_&_years.= sum(b25140be3, b25140be4, b25140ie3, b25140ie4, b25140he3, b25140he4, b25140ce3, b25140ce4, b25140fe3, b25140fe4, b25140ge3, b25140ge4, b25140de3, b25140de4, b25140ee3, b25140ee4);
+				nomortcstbrden_&_years.= sum(b25140be7, b25140ie7, b25140he7, b25140ce7, b25140fe7, b25140ge7, b25140de7, b25140ee7);
+				nomortsvrecstbrden_&_years. = sum(b25140be8, b25140ie8, b25140he8, b25140ce8, b25140fe8, b25140ge8, b25140de8, b25140ee8);
+				nomortcstbrdencalc_&_years.= sum(b25140be7, b25140be8, b25140ie7, b25140ie8, b25140he7, b25140he8, b25140ce7, b25140ce8, b25140fe7, b25140fe8, b25140ge7, b25140ge8, b25140de7, b25140de8, b25140ee7, b25140ee8);
+				allowncstbrden_&_years.= sum(b25140be3, b25140be7, b25140ie3, b25140ie7, b25140he3, b25140he7, b25140ce3, b25140ce7, b25140fe3, b25140fe7, b25140ge3, b25140ge7, b25140de3, b25140de7, b25140ee3, b25140ee7);
+				allownsvrecstbrden_&_years.= sum(b25140be4, b25140be8, b25140ie4, b25140ie8, b25140he4, b25140he8, b25140ce4, b25140ce8, b25140fe4, b25140fe8, b25140ge4, b25140ge8, b25140de4, b25140de8, b25140ee4, b25140ee8);
+				allowncstbrdencalc_&_years.= sum(b25140be3, b25140be4, b25140be7, b25140be8, b25140ie3, b25140ie4, b25140ie7, b25140ie8, b25140he3, b25140he4, b25140he7, b25140he8, b25140ce3, b25140ce7, b25140ce4, b25140ce8, b25140fe3, b25140fe7, b25140fe4, b25140fe8, b25140e3, b25140ge7, b25140ge4, b25140ge8, b25140de3, b25140de4, b25140de7, b25140de8, b25140ee3, b25140ee4, b25140ee7, b25140ee8);
+				rentcstbrden_&_years.= sum(b25140be11, b25140ie11, b25140he11, b25140ce11, b25140fe11, b25140ge11, b25140de11, b25140ee11);
+				rentsvrecstbrden_&_years.= sum(b25140be12, b25140ie12, b25140he12, b25140ce12, b25140fe12, b25140ge12, b25140de12, b25140ee12);
+				rentcstbrdencalc_&_years.= sum(b25140be11, b25140be12, b25140ie11, b25140ie12, b25140he11, b25140he12, b25140ce11, b25140ce12, b25140fe11, b25140fe12, b25140ge11, b25140ge12, b25140de11, b25140de12, b25140ee11, b25140ee12);
+
+
+				mortcstbrdenW_&_years.= b25140he3;
+				mortsvrecstbrdenW_&_years.= b25140he4;
+				mortcstbrdencalcW_&_years.= sum(b25140he3, b25140he4);
+				nomortcstbrdenW_&_years.= b25140he7;
+				nomortsvrecstbrdenW_&_years. = b25140he8;
+				nomortcstbrdencalcW_&_years.= sum(b25140he7, b25140he8);
+				allowncstbrdenW_&_years.= sum(b25140he3, b25140he7);
+				allownsvrecstbrdenW_&_years.= sum(b25140he4, b25140he8);
+				allowncstbrdencalcW_&_years.= sum(b25140he3, b25140he4, b25140he7, b25140he8);
+				rentcstbrdenW_&_years.= b25140he11;
+				rentsvrecstbrdenW_&_years.= b25140he12;
+				rentcstbrdencalcW_&_years.= sum(b25140he11, b25140he12)
+
```

Comment:
Missing semicolon at end of line above.

Reply:
Generated error message: 
```
NOTE 138-205: Line generated by the macro variable "_YEARS".
111              mortcstbrdenH_2019_23
                 _____________________
                 22
ERROR 22-322: Syntax error, expecting one of the following: !, !!, &, *, **, +, -, /, ;, <, <=, <>, =, >, ><, >=, AND, EQ, 
              GE, GT, IN, LE, LT, MAX, MIN, NE, NG, NL, NOTIN, OR, ^=, |, ||, ~=.  
```

This is typical of a missing semicolon error message where the _next_ statement is the one that gets flagged because SAS thinks it is part of the previous statement.